Sources/XcodeProj/Objects/Targets/PBXTargetDependency.swift

```diff
@@ -46,6 +46,10 @@ public final class PBXTargetDependency: PBXObject {
         }
     }
 
+    /// Platform filter attribute.
+    /// Introduced in: Xcode 11
+    public var platformFilter: String?
```

Comment:
Hey @TomasLinhart, thanks for adding this. Would you also mind extending the constructor with this attribute?